sources/components/custom-styles/template-cs-product.scss

```diff
@@ -0,0 +1,197 @@
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "700",
+      "off": "400"
+    },
+    "icon": "bold",
+    "scope": "global"
+  */
+  #{--product-title-font-weight}: 700;
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "font_style",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "italic",
+      "off": "normal"
+    },
+    "icon": "italic",
+    "scope": "global"
+  */
+  #{--product-title-font-style}: normal;
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "text_decoration",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "underline",
+      "off": "none"
+    },
+    "icon": "underline",
+    "scope": "global"
+  */
+  #{--product-title-font-text-decoration}: none;
+  /* VoogStyle
+    "pathI18n": ["product", "title"],
+    "titleI18n": "color",
+    "type": "button",
+    "editor": "colorPicker",
+    "featured": true,
+    "scope": "global"
+  */
+  #{--product-title-color}: $color-black;
+
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+  #{--product-price-size}: 24px;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "font_size",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "700",
+      "off": "400"
+    },
+    "icon": "bold",
+    "scope": "global"
+  */
+  #{--product-price-font-weight}: 700;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "font_style",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "italic",
+      "off": "normal"
+    },
+    "icon": "italic",
+    "scope": "global"
+  */
+  #{--product-price-font-style}: normal;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "text_decoration",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "underline",
+      "off": "none"
+    },
+    "icon": "underline",
+    "scope": "global"
+  */
+  #{--product-price-font-text-decoration}: none;
+  /* VoogStyle
+    "pathI18n": ["product", "price"],
+    "titleI18n": "color",
+    "type": "button",
+    "editor": "colorPicker",
+    "featured": true,
+    "scope": "global"
+  */
+  #{--product-price-color}: #000;
+  /* VoogStyle
+    "pathI18n": ["product", "description"],
+    "titleI18n": "size",
+    "editor": "rangePicker",
+    "min": 8,
+    "max": 100,
+    "unit": "px",
+    "scope": "global"
+  */
+  #{--product-description-size}: 20px;
+  /* VoogStyle
+    "pathI18n": ["product", "description"],
+    "titleI18n": "font_size",
+    "type": "button",
+    "editor": "toggleIcon",
+    "states": {
+      "on": "700",
+      "off": "400"
+    },
+    "icon": "bold",
+    "scope": "global"
+  */
+  #{--product-description-font-weight}: 400;
```

Comment:
The `font-weight` value of text content areas on the page is `100` – maybe change the description's font-weight as well?